<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Stack</class>
 <widget class="QMainWindow" name="Stack">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>457</width>
    <height>406</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QListWidget" name="listStack">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>90</y>
      <width>191</width>
      <height>271</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtTop">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>50</y>
      <width>161</width>
      <height>33</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>50</y>
      <width>31</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Top</string>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>50</y>
      <width>20</width>
      <height>261</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="Line" name="line_2">
    <property name="geometry">
     <rect>
      <x>422</x>
      <y>50</y>
      <width>20</width>
      <height>261</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>60</y>
      <width>161</width>
      <height>251</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Valor:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="txtValor"/>
     </item>
     <item>
      <widget class="QPushButton" name="btnPush">
       <property name="text">
        <string>Push</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnPop">
       <property name="text">
        <string>Pop</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnTop">
       <property name="text">
        <string>Top</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnClean">
       <property name="text">
        <string>Clean</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnClear">
       <property name="text">
        <string>Clear</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>457</width>
     <height>27</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuKenyStev">
    <property name="title">
     <string>KenyStev</string>
    </property>
   </widget>
   <addaction name="menuKenyStev"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
